apps/frontend/src/features/register/lanna/CreatingAccount.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { RegisterHeader } from "../../../components/ui/RegisterHeader";
+
+const CreatingAccount: React.FC = () => {
+  return (
+    <div className="flex flex-col grow">
+      <RegisterHeader
+        title="Setting up your account..."
+        subtitle="Share and connect effortlessly!"
+        description="With a simple tap, you can share your details and make connections instantly—without compromising your privacy. You’re always in control and can update or remove your info whenever you like."
+      />
+    </div>
+  );
+};
```

Comment:
Can you pass in some kind of delay on this page? it blipped on my screen for a fraction of a second when I registered a new devcon profile.